storage/tianmu/core/engine.cpp

```diff
@@ -710,6 +718,13 @@ AttributeTypeInfo Engine::GetAttrTypeInfo(const Field &field) {
       }
       throw common::UnsupportedDataTypeException();
     }
+    case MYSQL_TYPE_BIT: {
+      const Field_bit_as_char *f_bit = ((const Field_bit_as_char *)&field);
+      if (/*f_bit->field_length > 0 && */ f_bit->field_length <= common::TIANMU_BIT_MAX_PREC)
```

Comment:
here, we store the bit-value type in int64, and here why we throw this unspported exception.  as mysql does, it deals with bit-value type as bit string, so that the M should be in [1,64].

Reply:
As bit type is an unsigned numeric, in fact we should use `uint64_t` to store bit. but now the unsigned `int64_t` has not been supported, we use `int64_t` instead, thus the precision limited to [1, 63], we'll support `M=64`  in next milestone.